apps/nextjs-website/src/pages/[productSlug]/overview.tsx

```diff
@@ -13,28 +13,29 @@ export const getStaticPaths: GetStaticPaths<Params> = () => ({
 });
 
 export type OverviewPageProps = {
-  readonly product: Product;
-  readonly hero?: {
+  readonly hero: {
     readonly title: string;
     readonly subtitle: string;
   };
-};
+} & LayoutProps;
 
 export const getStaticProps: GetStaticProps<OverviewPageProps, Params> = ({
   params,
 }): GetStaticPropsResult<OverviewPageProps> => {
   const props = getOverview(params?.productSlug);
   if (props) {
-    return { props };
+    return { props: { ...props, products: getProducts().concat() } };
```

Comment:
```suggestion
    return { props: { ...props, products: [...getProducts()] } };
```
same for another comment